### ISSUE TITLE
parent_prefix and lookup

### ISSUE THREAD

**Issue by cancan101** [alanjds/drf-nested-routers#21]
Is there any documentation on what `parent_prefix` and `lookup` and mean/ what they should be set to on a `NestedSimpleRouter`?

**Comment by saneletm** [comment]
Once I nest my routers and get:
 http://host/api/parents/parentID/children/

Can I still do  ordering, search, filter on children?
    http://host/api/parents/parentID/children?search=<searchterm>

Is there special configuration I have to setup?

**Comment by jsenecal** [comment]
yes, with the apropriate configurations on the viewset/serializer

**Comment by saneletm** [comment]
do you have a sample appropriate config that I can see? How does this differ from doing un nested  for the viewset/serializer

**Comment by jsenecal** [comment]
No difference
